js/script.js

```diff
@@ -93,17 +93,17 @@ function getRandomQuote () {
   return quotes[getRandomQuote];
 }
 console.log(getRandomQuote());
```

Comment:
don't forget to remove ALL console.logs and debugger statements...before you merge this pull request